models/models.go

```diff
@@ -128,9 +128,29 @@ func (c Commodities) CustomsValue() (Money, error) {
 }
 
 type CompletedPackageDetails struct {
-	SequenceNumber string
-	TrackingIds    []TrackingID
-	Label          Label
+	SequenceNumber    string
+	TrackingIds       []TrackingID
+	Label             Label
+	OperationalDetail PackageOperationalDetail
+}
+
+type PackageOperationalDetail struct {
+	Barcodes Barcodes
+}
+
```

Comment:
This is fine for now. As I don't think that we are going to need to keep track of the type on specific objects I would consider changing Barcode to
```
type Barcodes struct {
     BinaryBarcodes []byte
     StringBarcodes string
}
```
And then writing a custom unmarshal function for the `Barcodes` struct

Reply:
Makes sense, but at the same time since we do not store the response, it wouldn't hurt to have the complete response, in a case in future we needed to do some checking on the type.